src/sentry/runner/commands/dev.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, print_function
+
+import click
+import os
+
+
+@click.group()
+def dev():
+    """
+    Development utilities.
+    """
+
+
+@dev.command()
+def psql():
+    """
+    Connect to Sentry's PostgreSQL in Docker.
+
+    Do not use in production!
+    """
+    os.execvp('/usr/local/bin/docker', (
+        '/usr/local/bin/docker',
+        'exec',
+        '-i',
+        '-t',
+        'sentry_postgres',
+        'bash', '-c', 'psql -U postgres sentry',
```

Comment:
Would there be an equivalent command in getsentry? Our current dev environment setup uses `getsentry` as the database name there. Could the database name come from `settings.DATABASES['default']['NAME']` instead?